generators/pkg/writer/write.go

```diff
@@ -0,0 +1,20 @@
+// Package writer provides a method to write the generated task
+// to the yaml document.
+package writer
+
+import (
+	"log"
+
+	v1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"sigs.k8s.io/yaml"
+)
+
+// WriteToDisk writes the generated task
+// to the []byte
+func WriteToDisk(task v1beta1.Task) ([]byte, error) {
```

Comment:
As is, this func is not adding additional benefit to callers.  When calling it, you would likely do something like:

```go
t := Task{...}
data, err := WriteToDisk(task)
if err != nil {
  return err
}
return data
```
which is near identical to the code within the func here. In most cases you would be able to just drop in replace it.

I suspect what we really want is to add more functionality to have it write to a file, which would make it different than just the yaml.Marshal. (this would also make the func name more fitting).